apps/fio/tests/data-integrity/test.yml

```diff
@@ -283,7 +287,7 @@
 
        - name: Check for error while writing
          shell: >
-           kubectl get logs {{ read_pod.stdout }} -n {{ app_ns }} | grep 'error caught' | wc -l
+           kubectl logs {{ read_pod.stdout }} -n {{ app_ns }} | grep 'error caught' | wc -l
```

Comment:
As we removed -f, it will search only once. Are you sure that the error can be caught at one instance?